src/test/resources/features/ifs/instalment_calculation/InstalmemtCalculationSingleDebt.feature

```diff
@@ -58,6 +57,56 @@ Feature: Instalment calculation for single debt - Input 2
       | 1                | 2023-04-02 | 100000    | 2538                      |
       | 5                | 2023-08-20 | 337592    | 1829                      |
 
+
+  @DTD-3163
+  Scenario: interestStartDate is included but not in the Future, then quote date should be used
+    Given debt instalment calculation with details
+      | duration | paymentFrequency | instalmentPaymentDate | interestCallDueTotal | numberOfDay | quoteType        | quoteDate  | isQuoteDateNonInclusive |
+      | 24       | monthly          | 2020-03-14            | 0                    | 1           | instalmentAmount | 2020-03-13 | false                   |
+    And the instalment calculation has no postcodes
+    And debt plan details with initial payment
+      | initialPaymentAmount | initialPaymentDate |
+      | 5000                 | 2020-03-13         |
+    And the instalment calculation has debt item charges
+      | debtId | debtAmount | mainTrans | subTrans | interestStartDate |
+      | 1234   | 100000     | 1545      | 1000     | 2025-01-14        |
+    When the instalment calculation detail is sent to the ifs service
+    Then IFS response contains expected values
+      | instalmentNumber | dueDate    | interestRate |
+      | 1                | 2020-03-13 | 3.25         |
+
+  @DTD-3163
+  Scenario: interestStartDate is included but in the Future, then interestStartDate should be used
+    Given debt instalment calculation with details
+      | duration | paymentFrequency | instalmentPaymentDate | interestCallDueTotal | numberOfDay | quoteType        | quoteDate  | isQuoteDateNonInclusive |
+      | 24       | monthly          | 2025-08-25            | 0                    | 1           | instalmentAmount | 2025-06-01 | false                   |
+    And the instalment calculation has no postcodes
+    And no initial payment for the debt item charge
+    And the instalment calculation has debt item charges
+      | debtId | debtAmount | mainTrans | subTrans | interestStartDate |
+      | 1234   | 100000     | 1545      | 1000     | DateInFuture      |
+    When the instalment calculation detail is sent to the ifs service
+    Then IFS response contains expected values
+      | instalmentNumber | dueDate    | interestRate |
+      | 1                | 2025-08-25 | 6.5          |
+
+  @DTD-3163
+  Scenario: With initial payment - interestStartDate is included but in the Future, then interestStartDate should be used
```

Comment:
Not sure if this additional scenario with the initial payment is necessary, does it test anything beyond what’s already covered?